src/animations/tweens.js

```diff
@@ -0,0 +1,54 @@
+import gsap from 'gsap';
+
+export const buttonAppearTween = button =>
```

Comment:
i think, that it woul be nice to call it here in this place as `buttonRef` - I assume, that ref is passed here.